saleor/graphql/attribute/types.py

```diff
@@ -89,13 +89,13 @@ class Attribute(CountableDjangoObjectType):
     slug = graphene.String(description=AttributeDescriptions.SLUG)
     type = AttributeTypeEnum(description=AttributeDescriptions.TYPE)
     unit = MeasurementUnitsEnum(description=AttributeDescriptions.UNIT)
-    values = FilterInputConnectionField(
+    choices = FilterInputConnectionField(
         AttributeValue,
         filter=AttributeValueFilterInput(),
         description=AttributeDescriptions.VALUES,
     )
 
-    value_required = graphene.Boolean(
+    choice_required = graphene.Boolean(
```

Comment:
I'm not sure if we should change this field. Required value can be true for attributes such as rich-text, not only those that have choices. E.g. if you have a product with rich-text attr, you can set "Value required" to true, which means that this rich-text will be required to save a product. This is a bit separate thing and I wouldn't change it unless necessary.

Reply:
Okey, I will change it back.